semantic_conventions/trace/messaging.yaml

```
@@ -31,6 +31,10 @@ groups:
     prefix: messaging.destination
     type: attribute_group
     brief: 'Semantic convention for attributes that describe messaging destination on broker'
+    note: >
+      Destination attributes SHOULD be set on publish, receive, or other span describing messaging messaging operation
+      if it operates with single message or if the attribute value applies to all messages in the batch.
```

```suggestion
      Destination attributes should be set when the messaging operation handles single messages. When the operation handles a batch of messages, the destination attributes should only be applied when the attribute value applies to all messages in the batch.
```

I feel this sentence was "merged" together with the above, so it was a bit confusing. I tried separating then so they can be read completely separately. 

I'm not sure about the `should` here, so I left it non-normative.